internal/wrappers/cgo/kuzzle/security.go

```diff
@@ -145,221 +131,332 @@ func kuzzle_security_destroy_role(r *C.role) {
 	C.free(unsafe.Pointer(r))
 }
 
-//export kuzzle_security_fetch_role
-func kuzzle_security_fetch_role(k *C.kuzzle, id *C.char, o *C.query_options) *C.role_result {
-	result := (*C.role_result)(C.calloc(1, C.sizeof_role_result))
+//export kuzzle_security_get_profile
+func kuzzle_security_get_profile(k *C.kuzzle, id *C.char, o *C.query_options) *C.profile_result {
+	result := (*C.profile_result)(C.calloc(1, C.sizeof_profile_result))
 	options := SetQueryOptions(o)
 
-	role, err := (*kuzzle.Kuzzle)(k.instance).Security.FetchRole(C.GoString(id), options)
+	profile, err := (*kuzzle.Kuzzle)(k.instance).Security.GetProfile(C.GoString(id), options)
 	if err != nil {
-		Set_role_result_error(result, err)
+		Set_profile_result_error(result, err)
 		return result
 	}
 
-	result.role = goToCRole(k, role, nil)
+	result.profile = goToCProfile(k, profile, nil)
 
 	return result
 }
 
-//export kuzzle_security_search_roles
-func kuzzle_security_search_roles(k *C.kuzzle, f *C.search_filters, o *C.query_options) *C.search_roles_result {
-	options := SetQueryOptions(o)
-	res, err := (*kuzzle.Kuzzle)(k.instance).Security.SearchRoles(cToGoSearchFilters(f), options)
-
-	return goToCRoleSearchResult(k, res, err)
+//export kuzzle_security_get_profile_rights
+func kuzzle_security_get_profile_rights(k *C.kuzzle, id *C.char, o *C.query_options) *C.json_result {
```

Comment:
We should not use json_result anymore but string_result as we use char* for json now